src/data/pages/users.ts

```diff
@@ -0,0 +1,190 @@
+import { sleep } from '../../services/utils'
+import { repeatArray } from '../../services/utils'
+import { User } from './../../pages/users/types'
+
+export const users = repeatArray<User>(50, [
```

Comment:
When user is updated - let's update it directly without triggering server reload of full list.

Reply:
I don't like this approach personally, but ok.

Reply:
@m0ksem share what you don't like then? :)

I think in almost all my projects there was a requirement for instant updates in list. And I can see why - it's very painful to change bunch of items quickly.